makroud.go

```diff
@@ -47,13 +47,16 @@ type Driver interface {
 	// Transaction
 	// ----------------------------------------------------------------------------
 
-	// Begin a new transaction.
+	// Begin begins a new transaction.
 	Begin() (Driver, error)
```

Comment:
@thoas I think we should break compatibility here, since almost every other methods needs a context.
_Example:_
- Exec
- Query
- Prepare

Reply:
The `Begin` method  of `sql.DB` doesn't require a context, not sure why we should introduce it since it won't be used?